frontend/src/components/bucket/BucketsTable.vue

```diff
@@ -53,16 +65,41 @@ const showInfo = async (id: number) => {
     </Column>
     <Column header="Actions" headerStyle="width: 12%" headerClass="header-right" bodyClass="content-right">
       <template #body="{ data }">
-        <Button class="p-button-lg p-button-rounded p-button-text"><font-awesome-icon icon="fa-solid fa-gear" /></Button>
+        <Button class="p-button-lg p-button-rounded p-button-text" @click="showPermissions(data.bucketId, data.bucketName)">
```

Comment:
Changed it to be 100px instead. Not sure what the ideal solution is. It will still squish if you make the viewport really small. But should work in the majority of situations.